src/qibolab/instruments/qblox/cluster_qcm_bb.py

```diff
@@ -151,162 +192,31 @@ def connect(self, cluster: QbloxCluster = None):
-                    target,
-                    "upsample_rate_awg_path0",
-                    "upsample_rate_awg_path1",
-                    value=0,
-                )
-
-            for target in [
-                self.device.sequencers[self.DEFAULT_SEQUENCERS["o1"]],
-                self.device.sequencers[self.DEFAULT_SEQUENCERS["o2"]],
-                self.device.sequencers[self.DEFAULT_SEQUENCERS["o3"]],
-                self.device.sequencers[self.DEFAULT_SEQUENCERS["o4"]],
-            ]:
-                self._set_device_parameter(
-                    target, "connect_out0", value="off"
-                )  # Default after reboot = True
-                self._set_device_parameter(
-                    target, "connect_out1", value="off"
-                )  # Default after reboot = True
-                self._set_device_parameter(
-                    target, "connect_out2", value="off"
-                )  # Default after reboot = True
-                self._set_device_parameter(
-                    target, "connect_out3", value="off"
-                )  # Default after reboot = True
-
-            self._set_device_parameter(
-                self.device.sequencers[self.DEFAULT_SEQUENCERS["o1"]],
-                "connect_out0",
-                value="I",
-            )
-            self._set_device_parameter(
-                self.device.sequencers[self.DEFAULT_SEQUENCERS["o2"]],
-                "connect_out1",
-                value="Q",
-            )
-            self._set_device_parameter(
-                self.device.sequencers[self.DEFAULT_SEQUENCERS["o3"]],
-                "connect_out2",
-                value="I",
-            )
-            self._set_device_parameter(
-                self.device.sequencers[self.DEFAULT_SEQUENCERS["o4"]],
-                "connect_out3",
-                value="Q",
-            )
-
-            # on initialisation, disconnect all other sequencers from the ports
-            self._device_num_sequencers = len(self.device.sequencers)
-            for sequencer in range(4, self._device_num_sequencers):
-                target = self.device.sequencers[sequencer]
-                self._set_device_parameter(target, "connect_out0", value="off")
-                self._set_device_parameter(target, "connect_out1", value="off")
-                self._set_device_parameter(target, "connect_out2", value="off")
-                self._set_device_parameter(target, "connect_out3", value="off")
+            self._set_default_values()
+            # then set the value loaded from the runcard
             try:
                 for port in self.settings:
                     self._sequencers[port] = []
                     self.ports[port].offset = self.settings[port]["offset"]
                     self.ports[port].hardware_mod_en = True
                     self.ports[port].nco_freq = 0
                     self.ports[port].nco_phase_offs = 0
-            except:
+            except Exception as error:
```

Comment:
Here we are catching a general `Exception` just to raise our own custom error. In principle we could just let Python/qblox raise their error directly and save some lines of code. The only advantage I see with raising a custom error is that we include the name of the failing module (I am not sure how readable their default errors are).

Reply:
I think that is useful including the name of the failing module, I've used this information may time for debugging purpose. Maybe in a stable version this will become useless but for now I prefer to keep it.